src/core/validation/ValidationRuleset.ts

```diff
@@ -45,7 +45,7 @@ export const createValidationRuleSet = ({
         }|maxNamespaceDuration`,
         namespaceName: {
             required: true,
-            regex: '^[a-z0-9]{1}[a-z0-9-_]{0,63}$',
+            regex: '^(?!symbol$)([a-z0-9]{1}[a-z0-9-_]{0,63})$',
```

Comment:
- symbol is not only reserved name (check node configuration https://github.com/nemtech/symbol-bootstrap/blob/4db690f095aa1e5ed14642ea1583264c1ac9d530/presets/shared.yml#L25)
- current error "The Namespace name field format is invalid" is not clear for user. 
Why we don't fix error message in rest?

Reply:
@cryptoBeliever that was a temp fix from wallet till we get rest updated, also all other reserved namespaces work as expected and don't need to be validated in client-side.
<img width="1279" alt="Screen Shot 2021-06-09 at 3 02 21 PM" src="https://user-images.githubusercontent.com/9143194/121359620-f5d7ae80-c933-11eb-9896-9ccb3cb500c0.png">

Reply:
@bassemmagdy 
I see. Could you link rest issue?

Reply:
https://github.com/nemtech/catapult-rest/issues/631

Reply:
Thanks. Maybe good idea is add comment in code that after fixing rest issue this condition can be removed.